apps/pie-docs/src/engineers/getting-started/structure.md

```diff
@@ -0,0 +1,11 @@
+---
+eleventyNavigation:
+    key: engineering-structure
+    title: Structure
+    parent: engineers-getting-started
+    order: 2
+---
+
+## Project Structure
+
+Coming soon…
```

Comment:
How come we aren't using the "coming soon" template here? Or are the changes super imminent?

Reply:
just to help out here I believe this is an imminent change :)

Reply:
Yep - they’ll be in my next PR